Sources/Wrap.swift

```diff
@@ -340,11 +344,13 @@ extension NSArray: WrapCustomizable {
 }
 
 /// Extension customizing how NSDictionaries are wrapped
-extension NSDictionary: WrapCustomizable {
-    public func wrap(context: Any?, dateFormatter: DateFormatter?) -> Any? {
-        return try? Wrapper(context: context, dateFormatter: dateFormatter).wrap(dictionary: self as [NSObject : AnyObject])
+#if os(OSX) || os(iOS) || os(watchOS) || os(tvOS)
```

Comment:
Good point. 😉  Let me know if additional changes are needed. Otherwise I'd be great if you could add a new tag/release, so that SPM can fetch the latest version.